### PR TITLE
update descriptions in Lexicons

### PR DESCRIPTION
**Pull request by bnewbold**
Probably need to do a codegen to update comments etc, but i'm not sure if that might cause merge conflicts with large feature branches.

Annotated many ambiguous endpoints as requiring auth (or not), but did not annotate all. Likewise annotated some as being implemented by specific network services, but didn't do this for every endpoint. This was taking a long time so I mostly just included stuff I was confident about, but might be ambiguous or confusing to devs.

Motivation here is to improve auto-generated docs.

Emily tagged for copy-editing and style, Devin for factual correctness.

### CODE REVIEW

File: lexicons/app/bsky/embed/images.json [hunk 1]
```diff
@@ -23,7 +23,10 @@
           "accept": ["image/*"],
           "maxSize": 1000000
         },
-        "alt": { "type": "string" },
+        "alt": {
+          "type": "string",
+          "description": "Alt-text description of the image, for accessibility."
```
Comment (emilyliu7321) [inline lexicons/app/bsky/embed/images.json:28]:
```suggestion
          "description": "Alt text description of the image, for accessibility."
```

File: lexicons/app/bsky/embed/images.json [hunk 2]
```diff
@@ -51,9 +54,18 @@
       "type": "object",
       "required": ["thumb", "fullsize", "alt"],
       "properties": {
-        "thumb": { "type": "string" },
-        "fullsize": { "type": "string" },
-        "alt": { "type": "string" },
+        "thumb": {
+          "type": "string",
+          "description": "Fully-qualified URL where a thumbnail of the image can be fetched. For example, CDN location provided by the AppView."
```
Comment (emilyliu7321) [inline lexicons/app/bsky/embed/images.json:59]:
```suggestion
          "description": "Fully-qualified URL where a thumbnail of the image can be fetched. For example, CDN location provided by the App View."
```

File: lexicons/app/bsky/embed/images.json [hunk 3]
```diff
@@ -51,9 +54,18 @@
...
+        },
+        "fullsize": {
+          "type": "string",
+          "description": "Fully-qualified URL where a large version of the image can be fetched. May or may not be the exact original blob. For example, CDN location provided by the AppView."
```
Comment (emilyliu7321) [inline lexicons/app/bsky/embed/images.json:63]:
```suggestion
          "description": "Fully-qualified URL where a large version of the image can be fetched. May or may not be the exact original blob. For example, CDN location provided by the App View."
```

File: lexicons/app/bsky/embed/images.json [hunk 4]
```diff
@@ -51,9 +54,18 @@
...
+        },
+        "alt": {
+          "type": "string",
+          "description": "Alt-text description of the image, for accessibility."
```
Comment (emilyliu7321) [inline lexicons/app/bsky/embed/images.json:67]:
```suggestion
          "description": "Alt text description of the image, for accessibility."
```

File: lexicons/app/bsky/embed/recordWithMedia.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "lexicon": 1,
   "id": "app.bsky.embed.recordWithMedia",
-  "description": "A representation of a record embedded in another form of content, alongside other compatible embeds.",
+  "description": "A representation of a record embedded in a Bluesky record (eg, a post), alongside other compatible embeds. For example, a quote-post and image, or a quote-post and external URL card.",
```
Comment (emilyliu7321) [inline lexicons/app/bsky/embed/recordWithMedia.json:4]:
```suggestion
  "description": "A representation of a record embedded in a Bluesky record (eg, a post), alongside other compatible embeds. For example, a quote post and image, or a quote post and external URL card.",
```

File: lexicons/app/bsky/feed/describeFeedGenerator.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
   "defs": {
     "main": {
       "type": "query",
-      "description": "Get information about a feed generator, including policies and offered feed URIs.",
+      "description": "Get information about a feed generator, including policies and offered feed URIs. Does not require auth; implemented by Feed Generator services (not AppView).",
```
Comment (emilyliu7321) [inline lexicons/app/bsky/feed/describeFeedGenerator.json:7]:
```suggestion
      "description": "Get information about a feed generator, including policies and offered feed URIs. Does not require auth; implemented by Feed Generator services (not App View).",
```

File: lexicons/app/bsky/feed/getFeed.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
   "defs": {
     "main": {
       "type": "query",
-      "description": "Get a hydrated feed from an actor's selected feed generator.",
+      "description": "Get a hydrated feed from an actor's selected feed generator. Implemented by AppView.",
```
Comment (emilyliu7321) [inline lexicons/app/bsky/feed/getFeed.json:7]:
```suggestion
      "description": "Get a hydrated feed from an actor's selected feed generator. Implemented by App View.",
```

File: lexicons/app/bsky/feed/getListFeed.json [hunk 8]
```diff
@@ -4,12 +4,16 @@
   "defs": {
     "main": {
       "type": "query",
-      "description": "Get a view of a recent posts from actors in a list.",
+      "description": "Get a feed of recent posts from a list (aka, posts and reposts from any actors on the list). Does not require auth.",
```
Comment (emilyliu7321) [inline lexicons/app/bsky/feed/getListFeed.json:7]:
```suggestion
      "description": "Get a feed of recent posts from a list (posts and reposts from any actors on the list). Does not require auth.",
```

**Review by emilyliu7321 — APPROVED** [review]
all minor copy tweaks

**Review by dholms — APPROVED** [review]
this is great, thanks for putting these together 🙌

**Comment by bnewbold** [comment]
@dholms when I went to rebase, codegen, and merge this, I got some scary diffs like:

```diff
@@ -167,39 +167,39 @@ export class Server {
 
 export class ComNS {
   _server: Server
-  atproto: ComAtprotoNS
+  atproto: AtprotoNS
```

is there some refactor in-flight?

This PR is docs-only, so i'm open to merging as-is; or if somebody else knows how to wrangle the codegen. It would be great to get this in by early next week, but also not a huge deal.

**Comment by mackuba** [comment]
See also: https://github.com/bluesky-social/atproto/issues/1713

**Comment by bnewbold** [comment]
Ooookey doke, figure out I was just missing `make build` and then codegen looks right. It's a doozy.

Also did a rebase before codegen.

closes: #1713

**Comment by dholms** [comment]
Ah yup sorry! We updated the lexgen cli & you probably out an out of date build

**Pull request merged**